Formula/expose-host-agent.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require_relative "../lib/gc/github_private_release_download_strategy"
+
+class ExposeHostAgent < Formula
+  desc "Wrapper command that exposes your local SSH agent for use during docker builds"
+  homepage "https://github.com/gocardless/expose-host-agent"
+  url "git@github.com:gocardless/expose-host-agent.git", :using => GitDownloadStrategy, :ref_type => :tag, :revision => "v0.0.1"
+  version "0.0.1"
+  sha256 "44418172f3803a988c4c114989259ffa7087298203afb7b7a3277fc30c65bb99"
+
+  depends_on "socat"
+
+  # Homebrew raises an exception if you don't install any files. We install a
```

Comment:
Also should the plist config still be here?

Reply:
1) Yep, deleting.
2) Nope, not running as a service any more! Chatted with @tscolari and we agreed that we'd rather only forward the agent when it's actually being used rather than all the time.